recipe/meta.yaml

```diff
@@ -6,79 +6,63 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nslsii-{{ version }}.tar.gz
   sha256: fc3e836ce70bee8f578881ae2df525349774951edfc0c1b8ab90ee0c4a777e57
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<37 or osx]
-  ignore_run_exports:
-    - python_abi
+  entry_points:
+    - sync-experiment = nslsii.sync_experiment:main
+    - what-is-ioc = nslsii.epics_utils:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 1
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
-    - python
+    - python >=3.6
     - appdirs
-    - bluesky
-    - bluesky-kafka  # [not win]
+    - bluesky >=1.8.1
+    - bluesky-kafka >=0.8.0
     - caproto
     - databroker
     - h5py
+    - httpx
     - ipython
     - ipywidgets
+    - ldap3
     - matplotlib-base
+    - msgpack-python >=1.0.0
+    - msgpack-numpy
     - numpy
     - ophyd
     - psutil
+    - pycryptodome
     - pyolog
     - redis-py
-    - tiled
+    - redis-json-dict
 
 test:
-  # requires:   # [not win]
-  #   - pip        # [not win]
-  # commands:   # [not win]
-  #   - pip check  # [not win]
   imports:
     - nslsii
-    - nslsii._version
-    - nslsii.ad33
-    - nslsii.common
-    - nslsii.common.ipynb
-    - nslsii.common.ipynb.animation
-    - nslsii.common.ipynb.info
-    - nslsii.common.ipynb.logutils
-    - nslsii.common.ipynb.nbviewer
-    - nslsii.detectors
-    - nslsii.detectors.trigger_mixins
-    - nslsii.detectors.utils
-    - nslsii.detectors.xspress3
-    - nslsii.detectors.zebra
-    - nslsii.devices
-    # The following error happened for Win/Py312:
-    #   AttributeError: module 'multiprocessing.connection' has no attribute 'CHALLENGE'. Did you mean: '_CHALLENGE'?
-    # Excluding the test for that platform/Python version for now.
-    - nslsii.iocs.eps_two_state_ioc_sim  # [not (win and py==312)]
-    # Uncomment the next line once https://github.com/NSLS-II/nslsii/issues/129
-    # is resolved.
-    # - nslsii.iocs.thermo_sim
-    - nslsii.temperature_controllers
-    - nslsii.transforms
```

Comment:
Some imports are still useful to capture potentially missing dependencies as `__init__` on the `nslsii` level may not have everything exposed. Can we revert those? I have a command which can generate the list: https://gist.github.com/mrakitin/b53d00d7ea167f3c7515f2929dd1199c.